Kernel/Arch/aarch64/Processor.h

```diff
@@ -168,6 +168,11 @@ class Processor {
         Aarch64::DAIF::set_I();
```

Comment:
Again the commit message (333723e544c51ab7370341515dcc09754743de3f) should describe better what is actually changed, because this commit does more than one thing.